build.gradle

```
@@ -16,7 +16,24 @@
  */
 
 group 'baritone'
-version '1.2.15'
+
+if (project.hasProperty("mod_version")) {
+    version project.mod_version
+} else {
+    def vers;
+    try {
+        vers = 'git describe --always --tags --dirty'.execute().text.trim()
+    } catch (IOException ignored) {
+        vers = ""
+    }
+    if (!vers.startsWith("v")) {
+        println "git doesn't appear to be installed!"
+        println "using version number: " + project.fallback_version
```

```suggestion
        println "git doesn't appear to be installed!"
    }
    if (!vers.startsWith("v")) {
        println "Failed to find a valid version tag using git"
        println "using fallback version number: " + project.fallback_version
```
Definitely move up the "not installed" message. The other two changes are optional.